### ISSUE TITLE
Search working badly - Only looks for the nickname and not for names

### ISSUE THREAD

**Issue by francrubio** [Proof-Of-Humanity/proof-of-humanity-web#192]
When using the search box, it only looks for the nickname of the people.
This is bad because you can't look for duplicates or even if you're looking for somebody you know.

For example:

I was looking for Agustin Canavas
https://app.proofofhumanity.id/profile/0x3150933b4a02dd9b47129e0ac48d352c918ffcc7?network=mainnet

If you look by the exact name, no results.
![image](https://user-images.githubusercontent.com/41177038/113809112-d5c21e00-973d-11eb-9f15-97b97841c49e.png)

if yo look for the name only "Agustin" , his profile did not pop up
![image](https://user-images.githubusercontent.com/41177038/113809184-f9856400-973d-11eb-8a1a-de4c604e8334.png)

**Comment by mtsalenc** [comment]
This is due to limitations on using subgraph. Unfortunately to have a decent search engine we'll need to implement a custom backend.

**Comment by indie-0x676f** [comment]
@francrubio It also seems like the search distinguishes between upper-case and lower-case characters?

**Comment by alcercu** [comment]
Yes, theGraph protocol uses GraphQL, and by the [specs](https://spec.graphql.org/June2018/#sec-Names) of the later one, searches are case-sensitive. A possible solution could be to create an additional field on the subgraph that stores names in lower case, and then we query against that field instead of name (converting the query to lower case too) 🤔 This could be seen as redundant data, though.
